### PR TITLE
Update server view

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/UnbindProjectDialog.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  @Override
+  protected void buttonPressed(int buttonId) {
+    if (buttonId == OK && !projects.isEmpty()) {
+
+      Job job = new Job(Messages.unbindProjectTask) {
+        @Override
+        protected IStatus run(IProgressMonitor monitor) {
+          try {
+            for (SonarLintProject project : projects) {
+              if (monitor.isCanceled()) {
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/UnbindProjectDialog.java]: MAJOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS134)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 2]
```diff
@@ -61,7 +61,9 @@ private Messages() {
   public static String viewInitializing;
 
   public static String actionDelete;
+  public static String actionUnbind;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 3]
```diff
@@ -76,6 +78,11 @@ private Messages() {
   public static String deleteServerDialogBoundProject;
   public static String deleteServerTask;
 
+  public static String unbindProjectDialogTitle;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 4]
```diff
@@ -61,7 +61,9 @@ private Messages() {
   public static String viewInitializing;
 
   public static String actionDelete;
+  public static String actionUnbind;
   public static String actionUpdate;
+  public static String actionUpdateBinding;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 5]
```diff
@@ -76,6 +78,11 @@ private Messages() {
   public static String deleteServerDialogBoundProject;
   public static String deleteServerTask;
 
+  public static String unbindProjectDialogTitle;
+  public static String unbindProjectDialogMessage;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 6]
```diff
@@ -76,6 +78,11 @@ private Messages() {
...
+  public static String unbindProjectDialogMessageMany;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java [hunk 7]
```diff
@@ -76,6 +78,11 @@ private Messages() {
...
+  public static String unbindProjectTask;
```
Comment (SonarTech) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)

**Comment by SonarTech** [comment]
SonarQube analysis reported 8 issues
* MAJOR 2 major
* MINOR 6 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ServerActionProvider.java#L120](https://github.com/SonarSource/sonarlint-eclipse/blob/5514b1f4e22f3477eb3461d78c992d986a940058/org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/server/ServerActionProvider.java#L120): The Cyclomatic Complexity of this method "fillContextMenu" is 15 which is greater than 10 authorized. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

**Comment by dbmeneses** [comment]
LGTM

**Pull request merged**